zubhub_frontend/zubhub/src/views/projects/Projects.jsx

```diff
@@ -28,6 +28,8 @@ import activity_of_the_month_svg from '../../assets/images/activity_of_the_month
 import activity_of_the_month_small_svg from '../../assets/images/activity_of_the_month_small.svg';
 import styles from '../../assets/js/styles/views/projects/projectsStyles';
 import commonStyles from '../../assets/js/styles';
+import upload from '../../assets/images/upload.svg';
```

Comment:
For visual consistency, maybe use this svg, Name it as `hiking.svg`:

![hiking_black_24dp](https://user-images.githubusercontent.com/3373857/163216370-626fb115-0bf9-476e-aae8-a4820af1a2aa.svg)

For the record, we've taken it from https://fonts.google.com/icons?icon.query=hiking

Reply:
rename `upload` as `hikingIcon`. Check references and update as well